docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -151,6 +162,7 @@ __Shorthand__: `-x`
 <p><small><span className="sauceGold">DEPRECATED</span></small></p>
 
 __Description__: Effective with Sauce Connect Proxy version 4.7.0, this flag was deprecated and replaced by [`--tunnel-pool`](#--tunnel-pool). Download the latest SC version [here](/secure-connections/sauce-connect/installation/).
+__Enviroment variable__: `SAUCE_NO_REMOVE_COLLIDING_TUNNELS`<br/>
```

Comment:
@waggledans , since this flag has been deprecated (per the badge), not sure if it's worth adding the env var info here.

cc @sweeneyskirt-sl

Reply:
Hmm... dunno... It's being deprecated (it will be supported for the next couple of years, most likely). U may be right, we don't want users to use this one